frontend/core-ui/src/modules/logs/components/filters/LogRecordTableFilterBars.tsx

```
@@ -0,0 +1,214 @@
+                    handleSelect={handleUserSelect}
+                  />
+                )}
+              />
+            </Popover.Content>
+          </Popover>
+          <Filter.BarClose filterKey="userIds" />
+        </Filter.BarItem>
+      )}
+
+      {customFilters.map(({ name, value }) => (
+        <Filter.BarItem>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)